node/forest_libp2p/src/behaviour.rs

```diff
@@ -336,4 +372,41 @@ impl ForestBehaviour {
     pub fn peers(&self) -> &HashSet<PeerId> {
         &self.peers
     }
+
+    /// Adds peer to bitswap peer set
+    pub fn connect(&mut self, peer_id: PeerId) {
+        self.bitswap.connect(peer_id);
+    }
+
+    /// Send a block to a peer over bitswap
+    pub fn send_block(
+        &mut self,
+        peer_id: &PeerId,
+        cid: Cid,
+        data: Box<[u8]>,
+    ) -> Result<(), Box<dyn Error>> {
+        debug!("send {}", cid.to_string());
+        let cid = cid.to_bytes();
+        let cid = Cid2::try_from(cid)?;
```

Comment:
Could be worth it implementing a try_from trait from cid into cid2?

Reply:
might be a bit weird to link the two cid crates together (as that's the only place that trait impl can be added)

I don't really care either way, just noting

Reply:
Yeah a local helper would be a good alternative, missed the part where they were in two different crates

Reply:
yeah I'm just thinking it would be nice to not link to the ipld core from our cid crate and just keep it in here.

Reply:
I agree. I do NOT want these conversions to be easy because they should be very rare. I hope these types of conversion should happen anywhere else.